<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorConfig</class>
 <widget class="QDialog" name="EditorConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑器选项</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/qdevcpp.ico</normaloff>:/qdevcpp.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>自动缩进</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="autoIndent">
         <property name="text">
          <string>　</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>显示空白</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="showWhiteSpace">
         <item>
          <property name="text">
           <string>不显示</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>所有</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>排除对齐</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>右边缘</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>启用</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="enableRightMargin">
           <property name="text">
            <string>　</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelMarginWidth">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>宽度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="marginWidth">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>自动保存</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>启用</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="enableAutoSave">
           <property name="text">
            <string>　</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSaveInterval">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>间隔时间</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="saveInterval">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>显示</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>高亮当前行</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>启用</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="highlightCurrent">
           <property name="text">
            <string>　</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCurrentColor">
           <property name="text">
            <string>颜色</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="currentColor">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="browseCurrentColor">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>字体</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;字体&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;字号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="fontSize">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFontComboBox" name="font">
           <property name="fontFilters">
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
           <property name="currentFont">
            <font>
             <family>Consolas</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>补全</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>括号补全</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="completeSBrace">
              <property name="text">
               <string>( )</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="completeMBrace">
              <property name="text">
               <string>[ ]</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="completeLBrace">
              <property name="text">
               <string>{ }</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="completeSQuote">
              <property name="text">
               <string>' '</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="completeDQuote">
              <property name="text">
               <string>&quot; &quot;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>enableRightMargin</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelMarginWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableRightMargin</sender>
   <signal>toggled(bool)</signal>
   <receiver>marginWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAutoSave</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelSaveInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAutoSave</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highlightCurrent</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCurrentColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highlightCurrent</sender>
   <signal>toggled(bool)</signal>
   <receiver>currentColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highlightCurrent</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseCurrentColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
